Test/git-issues/git-issue-1553.dfy

```diff
@@ -0,0 +1,10 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:cs "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:js "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:go "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:java "%s" >> "%t"
+// RUN: %diff "%s.expect" "%t"
+
+method Test<R>() {
+  var r: R;
+}
```

Comment:
Two other good test cases to add are:

``` dafny
method Test0<R(0)>() {
  var r: R;
}
method Test00<R(00)>() {
  var r: R;
}
```

These are probably unaffected by the fix, but they would still be good to include, to make sure they are working, too.

An additional way to extend these 3 test methods is to add

``` dafny
  var s: seq<R>;
  var t: array?<R>;
  var u: array<R>;
  var v: (R, int);
```

to the body of each of the three methods.